infra/modules/cognitive-services-docIntel.bicep

```diff
@@ -0,0 +1,56 @@
+param suffix string = uniqueString(resourceGroup().id)
+param managedIdentityPrincipalId string
+
+metadata description = 'Creates an Azure Cognitive Services instance.'
```

Comment:
```suggestion
metadata description = 'Creates an Azure Document Intelligence (form recognizer) instance.'
```

Reply:
would be nice replacing this with `Azure AI services` which enables multiple services with a single API key.